src/Makefile

```diff
@@ -29,6 +29,15 @@ else ifeq ($(TARGET),BLUE_PILL)
 	DEFS     = -DUSE_SYS3 -DFREE_STANDING -DBLUE_PILL
 	LDSCRIPT = stm32f103.ld
 	BOARD    = $(CHOPSTX)/board/board-blue-pill.h
+
```

Comment:
Please remove blank lines here and on line 40. There are no blank lines around other branches and there is no reason for this one to stand out :)